addons/website/models/website.py

```diff
@@ -241,11 +245,138 @@ def unlink(self):
         attachments_to_unlink.unlink()
         return super(Website, self).unlink()
 
-    def create_and_redirect_to_theme(self):
+    def create_and_redirect_configurator(self):
         self._force()
-        action = self.env.ref('website.theme_install_kanban_action')
+        action = self.env.ref('website.start_configurator_act_url')
         return action.read()[0]
 
+    # ----------------------------------------------------------
+    # Configurator
+    # ----------------------------------------------------------
+    def _iap_website_service(self, route, params):
+        website_service_endpoint = self.env['ir.config_parameter'].sudo().get_param('website.website_service_endpoint', DEFAULT_ENDPOINT)
```

Comment:
Why not split such long lines? use intermediary variable names, the code will be more readable and elegant.
You have similar long lines in many places. This often indicates that the line is doing too many things are once.